### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by Tumworobere**
**In this project, I achieved the following;**

   - Created models according to the given ERD diagram.
   - Implemented the authentication and authorization with devise.
   - Created the page to add a category.
   - Created the home page.
   - Created the page to add a transaction.
   - Created the transactions page.
   - Tested the app using RSpec.

### CODE REVIEW

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @Tumworobere, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 


## Required Changes ♻️
  - [ ] Some of your tests are still failing. Kindly address the issue and fix it.
  Screen Shot 2022-04-23 at 21 27 27


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] It is not in the project requirements, but It would be nice to fix the[ N+1 problem](https://medium.com/doctolib/understanding-and-fixing-n-1-query-30623109fe89)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: spec/integration/group_index_spec.rb [hunk 1]
```diff
@@ -3,25 +3,13 @@
...
+      @user1 = User.create! name: 'samara', password: 'samu@2022', email: 'samara@gmail.com'
       @group = @user1.groups.create! name: 'HBO', icon: 'icon'
       visit new_user_session_path
-      fill_in('Email', with: 'gbuldie@gmail.com')
-      fill_in('Password', with: 'godanah')
+      fill_in('Email', with: 'samara@gmail.com')
+      fill_in('Password', with: 'samu@2022')
       click_button('LOG IN')
       visit root_path
     end
```
Comment (mmsesay) [inline spec/integration/group_index_spec.rb]: - [ ] I will suggest you fix your tests. You still have some that are failing. The `group_index_spec.rb` is just one of them.
Please see the screenshot to see that you still have 6 failing tests that need retouching.

Test Screenshot |
:---------------------:|
a1

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @Tumworobere, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: spec/integration/group_show_spec.rb [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
   describe 'Group' do
     before(:each) do
-      @user1 = User.create! name: 'Gordon', password: 'godanah', email: 'gbuldie@gmail.com'
+      @user1 = User.create! name: 'samara', password: 'samu@2022', email: 'samara@gmail.com'
       @group = @user1.groups.create! name: 'HBO', icon: 'icon'
       visit new_user_session_path
-      fill_in('Email', with: 'gbuldie@gmail.com')
-      fill_in('Password', with: 'godanah')
+      fill_in('Email', with: 'samara@gmail.com')
+      fill_in('Password', with: 'samu@2022')
```
Comment (Gambit142) [inline spec/integration/group_show_spec.rb:10]: - [ ] I know you have tried working on the tests but currently six of them do not pass. This is the error I get when I run the command for the test

image

This is most likely from the way you are linking the buttons. Kindly fix this. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @Tumworobere 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sinansevgi — APPROVED** [review]
Hi @Tumworobere, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

[Optional]

- [ ] The Rails asset pipeline automatically loads both the CSS and SCSS files and cannot compile the view because you have two stylesheets with the same name (application.scss and application.css). This is why tests seem to fail. When I remove the `application.scss` file and try to run the tests, I can see that all tests are passing successfully. In order to prevent this from happening, you can remove one of the files.

Happy Hacking! :computer: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Sinan Sevgi

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Tumworobere** [comment]
> Hi @Tumworobere,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
> [Optional]
> 
>     * [ ]  The Rails asset pipeline automatically loads both the CSS and SCSS files and cannot compile the view because you have two stylesheets with the same name (application.scss and application.css). This is why tests seem to fail. When I remove the `application.scss` file and try to run the tests, I can see that all tests are passing successfully. In order to prevent this from happening, you can remove one of the files.
> 
> 
> Happy Hacking! 💻
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Sinan Sevgi
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you very much @sinansevgi  for this approval

**Pull request merged**